src/pages/user/libraries/[[...id]].tsx

```diff
@@ -1,18 +1,118 @@
-import { NextPage } from 'next';
+import { fetchLibraries, fetchLibraryEntity, librariesKeys, useGetLibraryEntity } from '@api';
+import { Center, Text } from '@chakra-ui/react';
+import {
+  CustomInfoMessage,
+  LibrariesLandingPane,
+  LibraryEntityPane,
+  LibrarySettingsPane,
+  LoadingMessage,
+  SimpleLink,
+} from '@components';
+import { composeNextGSSP } from '@ssr-utils';
+import { QueryClient } from '@tanstack/react-query';
+import { GetServerSideProps, NextPage } from 'next';
 import Head from 'next/head';
 
-const LibrariesHome: NextPage = () => {
+interface ILibrariesHomeProps {
+  id?: string;
+  subpage?: string;
+}
+
+const LibrariesHome: NextPage<ILibrariesHomeProps> = ({ id, subpage }) => {
+  const {
+    data: library,
+    isLoading: isLoadingLib,
+    error,
+    refetch,
+  } = useGetLibraryEntity(
+    {
+      id,
+    },
+    { enabled: !!id },
+  );
+
   return (
-    <div>
+    <>
       <Head>
-        <title>NASA Science Explorer - Libraries</title>
+        <title>NASA Science Explorer - Libraries - {!!id ? library?.metadata.name ?? '' : ''}</title>
```

Comment:
For dynamic titles we have to put the whole thing as a template string, otherwise it generates a `<!-- -->` comment string in the title